Plugins/DotNET/DotNETExports.cpp

```diff
@@ -176,27 +137,48 @@ void DotNET::StackPushVector(Vector value)
 
 void DotNET::StackPushEffect(CGameEffect* value)
 {
-    PushedCount += StackPushGameDefinedStructure(0, value);
+    LOG_WARNING("StackPushEffect is deprecated. Use StackPushGameDefinedStructure instead.");
```

Comment:
If we're deprecating it, we should update the managed code to match. I think we can make it a breaking change though and remove all these.